apps/ui/src/lib/isLocalLink.ts

```diff
@@ -0,0 +1,10 @@
+export function isLocalLink(link: string): boolean {
```

Comment:
The function is navigation-related so it should be placed in `navigation.ts` file

I would rename it to `isAppLink` because we are checking if link si routerable using next router or it is external (internet) link